src/EA.Weee.Requests/Scheme/TransferEvidenceNoteRequest.cs

```diff
@@ -43,6 +43,20 @@ public TransferEvidenceNoteRequest()
             EvidenceNoteIds = evidenceNoteIds;
         }
 
+        public TransferEvidenceNoteRequest(Guid organisationId,
+            Guid schemeId,
+            List<TransferTonnageValue> transferValues,
```

Comment:
no Condition for them, they could be null?!

Reply:
done... will add unit tests for this in another PR